src/blackmetal/src/communication/Client.hpp

```diff
@@ -15,11 +15,26 @@
 class Client
 {
 public:
+	/// Default constructor
```

Comment:
Netreba davat komentar, ved je to jasne, ze to je default constructor :)

Reply:
Ano ale je to sila zvyku